deployment/stacks/ui-auth.ts

```diff
@@ -151,7 +151,7 @@ export function createUiAuthComponents(props: CreateUiAuthComponentsProps) {
     scope,
     "CognitoIdentityPool",
     {
-      identityPoolName: `${stage}IdentityPool`,
+      identityPoolName: `${stage}-IdentityPool`,
```

Comment:
I'd make sure you deploy a stack and then deploy with this change and make sure it can be updated gracefully.

Reply:
tested and this was not an issue.